tests/unit/pillar/test_mysql.py

```diff
@@ -13,7 +13,7 @@
 
 
 @skipIf(NO_MOCK, NO_MOCK_REASON)
-@skipIf(PY3, 'MySQL-python is not compatible with python3')
+@skipIf(mysql.HAS_MYSQL is False, 'Install MySQL bindings before running MySQL unit tests.')
```

Comment:
It is easier to read in English "not has mysql" (understood as "has no mysql") instead of "has mysql is false" (understood as "the statement of having mysql is wrong"). And is shorter. So:
```python
@skipif(not mysql.HAS_MYSQL, 'Install.......')
```